subiquitycore/utils.py

```diff
@@ -68,10 +68,18 @@ def is_executable(x):
         for ftype, items in checks[check_type].items():
             for i in items:
                 if not os.path.exists(i):
-                    log.error('FAIL: {} is not found on the'
-                              ' filesystem'.format(i))
-                    env_ok = False
-                    continue
+                    if 'SNAP' in os.environ:
```

Comment:
I wonder if we can just delete the environment check now. Most other packages I use do not explicitly check their dependencies are present at runtime. But that's a change for another day I guess.